src/app/features/user/user.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { take } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { UserService } from './services/user.service';
+import { signIn } from '../../core/store/user/user.actions';
+
+
+@Component({
+  selector: 'app-user',
+  templateUrl: './user.component.html',
+  styleUrl: './user.component.scss'
+})
+export class UserComponent {
+  constructor(private userService: UserService, private readonly store: Store){}
+
+  public onSignUp(): void {
+    this.userService.signUp('man@nk.ru', '123456789').pipe(take(1)).subscribe(res => console.log(res))
```

Comment:
Maybe we should make comments for test code? So that there will be no questions later on.
For example:
```
TEST: Checking the work of the services method "signUp"

Reply:
I can delete it from user component. German told I'll continue work with this component during next train and I thougth this code can be left here.